plugins/modules/consul.py

```diff
@@ -31,9 +32,10 @@
  - "See U(http://consul.io) for more details."
 requirements:
   - python-consul
-  - requests
 author: "Steve Gargan (@sgargan)"
 extends_documentation_fragment:
+  - community.general.consul.documentation
```

Comment:
```suggestion
  - community.general.consul
```
You don't have to specify `.documentation`, it's the default.

(I guess it's a matter of style; most don't specify it, but some do. Decide for yourself ;) )